edfio/edf.py

```diff
@@ -985,6 +1005,22 @@ def drop_signals(self, drop: Iterable[int | str]) -> None:
         )
         self._num_signals = Edf.num_signals.encode(len(selected))
 
+    def append_signals(self, new_signals: EdfSignal | Iterable[EdfSignal]) -> None:
```

Comment:
We could also add an `insert_signal(self, new_signal: Edfsignal, idx: int)` method and a `replace_signal(self, new_signal: EdfSignal, replace_signal: int | EdfSignal | str)` method.

Reply:
Great idea! I'll create a new issue for that to let us merge this PR quickly.